config/core/test-configs.yaml

```diff
@@ -2658,6 +2679,7 @@ test_configs:
       - baseline
       - cros-ec
       - igt-kms-tegra
+      - kselftest-alsa
```

Comment:
Please drop this line, this platform doesn't have the capacity to run anything more.